kalite/testing/behave_helpers.py

```diff
@@ -47,7 +47,7 @@
 
 
 # Maximum time to wait when trying to find elements
-MAX_WAIT_TIME = 1
+MAX_WAIT_TIME = 5
```

Comment:
It's only a problem for Python 2.6, waiting for a new page to load just takes longer on Travis. The wait time is only a problem if it *needs* to wait that long, if the condition is satisfied sooner it won't wait that long.